.rubocop.yml

```diff
@@ -2,32 +2,53 @@ AllCops:
   Exclude:
     - 'bin/**/*'
     - 'db/**/*'
-Rails:
-  Enabled: true
+  TargetRubyVersion: 2.3
+
 Metrics/LineLength:
   Enabled: false
+
+Rails:
+  Enabled: true
+
 Style/CommentAnnotation:
   Enabled: false
+
 Style/Documentation:
   Enabled: false
+
 Style/DotPosition:
   Enabled: false
+
 Style/EmptyElse:
   Enabled: false
-Style/GuardClause:
+
+Style/FrozenStringLiteralComment:
   Enabled: false
+
+Style/GuardClause:
+  Enabled: true
+
 Style/IfUnlessModifier:
-  # inline conditionals don't get picked up by simplecov, so preferable to avoid them
-  Enabled: false
+  Description: >-
+                 Favor modifier if/unless usage when you have a
+                 single-line body.
+  StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier'
+  Enabled: true
+
 Style/RedundantBegin:
   Enabled: false
+
 Style/RedundantSelf:
   Enabled: false
+
 Style/SignalException:
   Enabled: false
+
 Style/StringLiterals:
   EnforcedStyle: double_quotes
+
 Style/AndOr:
-  Enabled: false
+  EnforcedStyle: always
```

Comment:
We had disabled checking for inline if's because of a simplecov failure to count them properly.